navigation/src/androidMain/kotlin/com/freeletics/khonshu/navigation/NavEventNavigator.kt

```diff
@@ -66,6 +66,16 @@ public interface Navigator {
      */
     public fun resetToRoot(root: NavRoot)
 
+    /**
+     * Replace the current back stack with the given [root].
+     * The current back stack will cleared and the given [root] will be recreated.
+     * After this call the back stack will only contain the given [root].
+     *
+     * This differs from [resetToRoot] in that [resetToRoot] does not pop the start route (exclusive)
+     * whereas this does.
+     */
+    public fun replaceAll(root: NavRoot)
```

Comment:
Could you add something for this to the test helper https://github.com/freeletics/khonshu/blob/main/navigation-testing/src/androidMain/kotlin/com/freeletics/khonshu/navigation/NavigatorTurbine.kt

Reply:
Done https://github.com/freeletics/khonshu/pull/657/commits/00dd521c1c72a39ec9c054c5098386bd101a3af1